tensorflow/python/estimator/inputs/queues/feeding_functions.py

```diff
@@ -21,6 +21,8 @@
 import collections
 import random
 import numpy as np
+import six
+from types import FunctionType
```

Comment:
done. I use `import types as tp` because `types` is used as a variable inside the function